modules/ocf/files/packages/cups/printers.conf

```diff
@@ -0,0 +1,41 @@
+# Printer configuration file for CUPS v2.2.1
```

Comment:
How much of this custom? If it's only a couple lines we can use `augeas` on the config that cups pulls in.

Reply:
Turns out, most of the lines are custom... since upon installation, CUPS starts out with no printers... so I suppose I might as well just include the entire file